Simplenote.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 52;
+	objectVersion = 54;
```

Comment:
This and the `Xcode 12.0` change below are so that CocoaPods can keep using a file to provide the frameworks list as part of its integration. Otherwise, it will list them directly in the project, making for noisier diffs.